hydra-node/json-schemas/api.yaml

```diff
@@ -806,7 +806,22 @@ components:
                   },
                   "redeemer": "21"
               }
-            }
+            },
+            "0700070603080308050807020202020806080000030605030704010503040505#10": {
+                "address": "addr_test1xrfd3ey8p85wcx8rhk8hxtrdgw580ukfsa9f2kqs7hpw5hkmyp5cm0cvpt6hx5vd34fvjtk07uxlkw6f8g0ewamhwuds9p7e97",
+                "datum": null,
```

Comment:
We should not serialize `null` fields. Not being present is preferred as it's less verbose IMO. WDYT?

Reply:
I don't have a preference but what you point here should be the json of our own `UTxO'` for which we use newtype deriving strategy. I would keep this decision separate from this PR and add custom instances for whatever type we don't want to see null fields in json. wdyt?